frontend/src/lib/CBRequest.ts

```diff
@@ -36,11 +78,57 @@ const getRequest = async (endpoint: AllowedEndpoints, options: RequestArgs) => {
     };
   }
 
-  const url = `${BACKEND_URL}${endpoint}`;
+  const url = generateUrl(endpoint, options.params);
   const { headers } = options;
 
   const response = await axios.get(url, { headers });
   return response.data;
 };
 
-export { postRequest, getRequest };
+const prepareImageData = (
+  selectedImage: ImagePickerAsset,
+  data: Record<string, any>,
+): FormData => {
+  const uri =
+    Platform.OS === "android"
+      ? selectedImage.uri
+      : selectedImage.uri.replace("file://", "");
+  const filename = selectedImage.uri.split("/").pop();
+  const match = /\.(\w+)$/.exec(filename as string);
```

Comment:
Why are we splitting out the extension and readding it again, should we leave it as is?